src/views/GameView.vue

```diff
@@ -1052,19 +1052,19 @@ export default {
   position: absolute;
 }
 // slide-below (enter and leave below)
-.slide-below-enter,
+.slide-below-enter-from,
```

Comment:
Vue3 changed the initial frame class to .enter-from: https://vuejs.org/guide/built-ins/transition.html#css-based-transitions

Reply:
Ahhhh I knew it! Good stuff!